modules/_labs/badge/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,73 @@
+/// <reference path="../../../../../typings.d.ts" />
+import React from 'react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {Table} from '../../../../../utils/storybook';
+
+import Badge from '../index';
+import README from '../README.md';
+
+export default {
+  title: 'Labs|Badge/React/',
+  decorators: [withReadme(README)],
+  parameters: {
+    chromatic: {
+      disable: false,
+    },
+  },
+};
+
+export const VisualTesting = () => {
+  return (
+    <Table>
+      <thead>
+        <tr>
+          <th></th>
+          <th>Empty</th>
+          <th>Single Digit</th>
+          <th>Double Digit</th>
+          <th>Triple Digit</th>
+          <th>Greater Than 999</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Default</td>
+          <td>
+            <Badge aria-label={`new unread notifications`} />
+          </td>
+          <td>
+            <Badge aria-label={`${1} unread notifications`} count={1} />
+          </td>
+          <td>
+            <Badge aria-label={`${23} unread notifications`} count={23} />
+          </td>
+          <td>
+            <Badge aria-label={`${999} unread notifications`} count={999} />
+          </td>
+          <td>
+            <Badge aria-label={`${1000} unread notifications`} count={1000} />
+          </td>
+        </tr>
+        <tr>
+          <td>Inverse</td>
+          <td>
+            <Badge aria-label={`new unread notifications`} variant="inverse" />
+          </td>
+          <td>
+            <Badge variant="inverse" aria-label={`${1} unread notifications`} count={1} />
+          </td>
+          <td>
+            <Badge variant="inverse" aria-label={`${23} unread notifications`} count={23} />
+          </td>
+          <td>
+            <Badge variant="inverse" aria-label={`${999} unread notifications`} count={999} />
+          </td>
+          <td>
+            <Badge variant="inverse" aria-label={`${1000} unread notifications`} count={1000} />
+          </td>
+        </tr>
+      </tbody>
+    </Table>
```

Comment:
Nit: This would be much simpler if you used `ComponentStatesTable`. No need to export `Table` in that case

```suggestion
      <ComponentStatesTable
        rowProps={[
          {label: 'Default', props: {}},
          {label: 'Inverse', props: {variant: Badge.Variant.Inverse}},
        ]}
        columnProps={[
          {label: 'Empty', props: {}},
          {label: 'Single Digit', props: {count: 1}},
          {label: 'Double Digit', props: {count: 23}},
          {label: 'Triple Digit', props: {count: 999}},
          {label: 'Greater than 999', props: {count: 1000}},
        ]}
      >
        {props => (
          <Badge {...props} aria-label={`${props.count} unread notifications`} />
        )}
      </ComponentStatesTable>
```

Reply:
Yeah, I thought the `ComponentStatesTable` might be misleading as Badge is non-interactive? No hover / focus states.

Reply:
Since this is for visual testing and not really for consumers viewing, I think it's preferable to use the more concise option.

Also just because they're not interaction states doesn't mean these aren't still states (empty state, overcount state, etc.)